functions/configure.ts

```diff
@@ -102,7 +102,9 @@ export default SlackFunction(def, async ({ inputs, client, env }) => {
         }
       } catch (e) {
         console.log(e);
-        modalMessage = e;
+        modalMessage =
+          "*:warning: Apologies! Failed to configure this app due to the following error:*\n>" +
```

Comment:
Passing raw string didn't work in this scenario. So, I've changed how to display it as a block on a modal